### ISSUE TITLE
--os=X.Y not respected

### ISSUE THREAD

**Issue by n0brainer** [grahampugh/erase-install#40]
I've been trying to update the scripts to the current versions, and it seems it's not respecting --os=10.14 nor --version=10.14.6

![Screen Shot 2020-10-22 at 2 23 17 PM](https://user-images.githubusercontent.com/2116991/96919944-2553ac80-1472-11eb-916a-80376e1e13d2.png)

`Script result: 
   [erase-install] Script execution started: Thu Oct 22 14:20:38 CDT 2020
   [erase-install] Caffeinating this script (pid=20591)
   [erase-install] Looking for existing installer
   [find_existing_installer] Installer found at /Applications/Install macOS Mojave.app.
   [check_installer_is_valid] Checking validity of /Applications/Install macOS Mojave.app.
   [check_installer_is_valid] 18G101 < 19H2 so not valid.
   [erase-install] ERROR: Invalid installer is present. Run with --overwrite option to ensure that a valid installer is obtained.`

**Comment by grahampugh** [comment]
Hi, that is expected behaviour. The script will use an existing installer if it finds one. It doesn't really check what version it is. But if you use the --overwrite option it will delete the existing installer and download the one you want.

**Comment by n0brainer** [comment]
I cached the installer in applications 
, it sees it’s there and confirms its 10.14.6. I set the switch to 10.14.6 (shown above), but the script still tries to force 10.15.7. 

Truth be told I was trying to run it on a 10.15 machine. I tried it again on a 10.13.6 machine and it did wipe and reload to 10.14.6 (that I had cached). So maybe it’s just not wanting to go backwards, even though the machine is 10.14 capable.
